deploy/chart/templates/0000_90_olm_01-prometheus-rule.yaml

```
@@ -18,6 +18,8 @@ spec:
           severity: warning
           namespace: "{{ "{{ $labels.namespace }}" }}"
         annotations:
+          summary: CSV failed for over 2 minutes
+          description: Fires whenever a CSV has been in a state other than succeeded for more than 2 minutes
```

Apologies, I had not meant to submit that review comment yet.
```suggestion
          summary: CSV failed for over 2 minutes.
          description: Fires whenever a CSV has been in the failed phase for more than 2 minutes.
```